client/app/lib/onboarding.coffee

```diff
@@ -207,7 +207,8 @@ module.exports = class Onboarding
 
     # Trigger a 'done' pseudo-event, corresponding to onboarding end.
     triggerDone: ()->
-        throw new Error 'Not implemented yet'
+        # redirection to the cozy home
+        window.location.replace '/'
```

Comment:
Totally agree with you on this point. I changed this part to use this better solution. Thanks @gregorylegarec.